### PR TITLE
ML#119 - Investigate Localised Field Settings

### PR DESCRIPTION
**Pull request by kodinkat**
- fixes: #119

### CODE REVIEW

File: magic-link/magic-link-templates.php [hunk 1]
```diff
@@ -295,7 +295,7 @@ private function localized_selected_field_settings( $template ) {
         if ( !empty( $template['fields'] ) ) {
 
             $localized_selected_field_settings = [];
-            foreach ( $template['fields'] ?? [] as $template_field ) {
+            foreach ( $template['fields'] as $template_field ) {
```
Comment (corsacca) [inline magic-link/magic-link-templates.php:298]: @kodinkat, feels like that `?? []` should be kept. No?

**Comment by corsacca** [comment]
@kodinkat lets filter the initial "post" field
And the ones returned by the api too.
image

File: magic-link/magic-link-templates.php (hunk 1 above)
Comment (kodinkat) [reply magic-link/magic-link-templates.php:298]: Empty if statement check on ln:295 confirms fields array exists...

Comment (corsacca) [reply magic-link/magic-link-templates.php:298]: 👍

**Pull request merged**